features/admin/invite_supplier_contributor.feature

```diff
@@ -1,14 +1,14 @@
 @admin @invite-supplier-contributor
 Feature: Invite a contributor to a supplier account
 
-@requires-credentials @notify
+@requires-credentials @notify @skip-staging
 Scenario Outline: Correct users can invite a contributors to a supplier account
   Given I am logged in as the existing <role> user
   And I have a supplier with:
     | name          | DM Functional Test Supplier - Invite a contributor feature |
   And I click the 'Edit supplier accounts or view services' link
   And I enter 'DM Functional Test Supplier - Invite a contributor feature' in the 'supplier_name' field and click its associated 'Search' button
-  And I click a summary table 'Users' link for 'DM Functional Test Supplier - Invite a contributor feature'
+  And I click the summary table 'Users' link for the 'DM Functional Test Supplier - Invite a contributor feature' link
   When I enter 'simulate-delivered@notifications.service.gov.uk' in the 'Email address' field
```

Comment:
I know this isn't something you've changed, but this too could do with being one of the 3 "special" domains...

Reply:
Will do